fpga/mcs.wake

```diff
@@ -19,6 +19,61 @@ def vc707migHook =
   def addResources = "vivado/vc7071gmig/1.0"
   makeBlackBoxHook name addResources
 
+global target fpgaSynthOnly plan =
```

Comment:
The issue with this, and why I made it completely separate, is that the only return value I care about is the error status of the vivado job. So I essentially want to run pretty much the normal vivado job (with a different top tcl file) but not do anything with the outputs. The reason is that I want to use this as a form of FPGA CI, but I want it to just run through the beginning of the process and see if it errors or not

Reply:
And I don't know of a way to refactor the wake code such that a job can have varying return types, so I just copy-pasted

Reply:
Just wanted to open up my original request of renaming the function to be a verb, since it actually does something. Otherwise, this reads like a configuration option.

Reply:
Yes, sorry I forgot that. changed it to `runFPGASynthOnly`